### PR TITLE
Add Mock VSCode API for Pages

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30767

**Changes**
- Add Mock VS code API data for pages so that pages can be retrieved when the application is run in the browser.

**How to Check**
- Navigate to `src/client`
- Run `yarn start`
- Go to `localhost:3000`
- Navigate through the wizard
- You should be able to see the pages

**Note**: The pages have been defaulted to React pages.

### CODE REVIEW

**Review by SahilTara — DISMISSED** [review]

File: src/client/src/mockData/mockVsCodeApi.ts [hunk 1]
```diff
@@ -212,6 +212,138 @@ const mockVsCodeApi = () => ({
...
+                    name: "Blank",
+                    defaultName: "Blank",
+                    description:
+                      "A blank page for you to build your web application from scratch.",
+                    richDescription:
+                      "This is the most basic page. A blank canvas to mold into whatever you wish. The blank page leaves pretty much everything up to you.",
+                    author: "Microsoft",
+                    version: "1.0.0",
+                    icon:
+                      "C:Users\t-nijindAppDataLocalCoreTSTemplatesLocalEnv\0.0.0.0WebPagesReact.Blank.template.configicon.xaml",
```
Comment (SahilTara) [inline src/client/src/mockData/mockVsCodeApi.ts]: Just noticed this, may want to change the icon to some dummy text, as the extension handles it. Happens in many places.

Comment (jsondoo) [reply src/client/src/mockData/mockVsCodeApi.ts]: We changed this to empty strings in the mockvscode api for the frameworks icon if u wanna do that @njindal239

Comment (streetcoder123) [reply src/client/src/mockData/mockVsCodeApi.ts]: Makes sense! Thanks for that :)

**Review by SahilTara — APPROVED** [review]

**Review by jcoc611-microsoft — APPROVED** [review]
Nice!

**Review by jasmingoh — APPROVED** [review]

**Review by piotrmark — APPROVED** [review]

**Pull request merged**